### ISSUE TITLE
Autoresize Textarea

### ISSUE THREAD

**Issue by SuperDOS** [beercss/beercss#276]
Would it be possible to resize the textarea inputs similiar to what materialize-css does?

**Comment by leonardorafael** [comment]
Yes we can do that. Today the JS file of Beer CSS is optional and if you use it, you can do the job with almost ZERO JS in your project. We will try to add this on next release.

**Comment by guidouil** [comment]
I would love that to for a chat interface with AI ;)

**Comment by SuperDOS** [comment]
noticed now when version 123 of chrome is released you can use the new css attribute field-sizing:
textarea {
    field-sizing: content;
  }
